resolver/src/lookup.rs

```diff
@@ -30,7 +30,7 @@ use resolver_future::BasicResolverHandle;
 
 /// Result of a DNS query when querying for any record type supported by the TRust-DNS Client library.
```

Comment:
> TRust-DNS Client library

Since your cleaning up comments, want to fix the reference from Client to Resolver? :)